libs/anglify/src/modules/snackbar/_variables.scss

```
@@ -0,0 +1,3 @@
+$snackbar-background-color: var(--anglify-snackbar-background-color, var(--color-background));
+$snackbar-container-color: var(--anglify-snackbar-container-color, var(--color-on-surface-high-emphasis));
```

How about we add --color-surface-inverted globally and use that color here instead of the color intended for fonts?

```suggestion
$snackbar-container-color: var(--anglify-snackbar-container-color, var(--color-surface-inverted));
```
```scss
--color-surface-inverted: #000000;
```

---

Not sure, its not in the spec and the spec itself uses `on-surface-high-emphasis`, so I'm not sure its intended for fonts honestly.

I think it just denotes that the color is to be used ON TOP of the surface color.